tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/Exchange.scala

```diff
@@ -79,6 +79,7 @@ object Exchange {
     val session = SparkSession
       .builder()
       .appName(PROGRAM_NAME)
+      .master("local")
```

Comment:
if we specified the master as local could we config it from command line?

Reply:
Sorry, I forget to delete the test code.